lib/Backend/arm64/LowerMD.cpp

```diff
@@ -1186,7 +1186,7 @@ LowererMD::LowerEntryInstr(IR::EntryInstr * entryInstr)
     ULONG stackAllocation1 = (layout.TotalStackSize() < 512) ? layout.TotalStackSize() : layout.RegisterAreaSize();
     ULONG stackAllocation2 = layout.TotalStackSize() - stackAllocation1;
 
-//    this->GenerateDebugBreak(insertInstr);
+    this->GenerateDebugBreak(insertInstr);
```

Comment:
>GenerateDebugBreak [](start = 10, length = 18)

ha we don't want this :-)

Reply:
Caught it just before you were commented. ;)